src/styled/customStyles.scss

```diff
@@ -14,4 +14,18 @@
       top: auto;
     }
   }
+}
+
+// bootstrap does not natively have this class
+.cursor-pointer {
```

Comment:
any reason we need this where not using an `a` or `button` would work since they come with this style built-in?

Reply:
discovered they are not built in
https://stackoverflow.com/questions/47269189/bootstrap-v4-css-class-for-pointer-cursor

Reply:
```
UPDATE for Bootstrap 4 stable
The cursor: pointer; rule has been restored, so buttons will now by default have the cursor on hover:

<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
<button type="button" class="btn btn-success">Sample Button</button>
```

Reply:
i think we can replace the `Add Card` component with just a `<Button />` component and use  that.  that way we can avoid using an `a` tag for non page links and also get the same functionality

Reply:
replaced with `<Button />` for consistency